apps/richie/templates/richie/dc.yml.j2

```diff
@@ -6,20 +6,20 @@ metadata:
     service: richie
     version: "{{ richie_image_tag }}"
     deployment_stamp: "{{ deployment_stamp }}"
-  name: richie-{{ deployment_stamp }}
+  name: "richie-richie-{{ deployment_stamp }}"
```

Comment:
Arf... Remember when I told you that the service should be called "app" and not richie. I think you really need to consider that. And it's more natural to follow the docker-compose semantic...

Reply:
Ok, you are right: let's do this.